downstream/modules/platform/proc-aap-create_eda.adoc

```diff
@@ -0,0 +1,19 @@
+[id="aap-create_eda"]
+
+= Creating an EDA object
+
+[role=_abstract]
+
+Use the following steps to create an *EDA* custom resource object.
+
+.Procedure
+. Log in to *{OCP}*.
+. Navigate to menu:Operators[Installed Operators].
+. Select the {OperatorPlatformNameShort} installed on your project namespace.
+. Select the *Automation Hub* tab.
+. Click btn:[Create AutomationHub]. You can create the object through the *Form view* or *YAML view*. The following inputs are available through the *Form view*.
+. Enter a name for the new deployment.
+. In *Advanced configurations*:
```

Comment:
Are these part of the Form view? If they are, then maybe they should be indented